scripts/upload_incr_DB.py

```diff
@@ -385,12 +403,13 @@ def main():
 	tt = int(args['txblktime'])
 	dd = int(args['dsblktime'])
 	webhook = args['webhook']
+	source = os.path.dirname(os.path.abspath(__file__)) + "/"
 
 	# create temp folder
-	if not os.path.exists(SOURCE+'temp'):
-		os.makedirs(SOURCE+'temp')
-	CleanupDir(SOURCE+'temp')
-	os.chdir(SOURCE)
+	if not os.path.exists(source+'temp'):
```

Comment:
Can we use the [standard library](https://docs.python.org/3.7/library/tempfile.html) to handle the temporary directory? If the folder is really meant to be temporary and not meant to be permanent, it should be recycled automatically by the code or by the system (if you are using `/tmp/` path)

Reply:
It's a good suggestion and I think it's better to have a separated PR to refactor the `temp` thing. @sandipbhoir